pipeline/4/Jenkinsfile-tier-1-cephfs.groovy

```diff
@@ -0,0 +1,50 @@
+/*
+    Pipeline script for executing Tier 1 Cephfs test suites for RH Ceph 4.x
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def cephVersion = "nautilus"
+def sharedLib
+
+// Pipeline script entry point
+
+node(nodeName) {
+
+    timeout(unit: "MINUTES", time: 30) {
+        stage('Install Prereq') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: '*/master']],
+                doGenerateSubmoduleConfigurations: false,
+                extensions: [[
+                    $class: 'SubmoduleOption',
+                    disableSubmodules: false,
+                    parentCredentials: false,
+                    recursiveSubmodules: true,
+                    reference: '',
+                    trackingSubmodules: false
+                ]],
+                submoduleCfg: [],
+                userRemoteConfigs: [[
+                    url: 'https://github.com/red-hat-storage/cephci.git'
+                ]]
+            ])
+            sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+            sharedLib.prepareNode()
+        }
+    }
+
+    timeout(unit: "MINUTES", time: 360) {
```

Comment:
The timeout needs to be controlled from the wrapper job... not individual ones... Would it be possible to remove this tag from here?

Reply:
Sure, will do that